airflow/triggers/external_task.py

```diff
@@ -38,25 +39,33 @@ class TaskStateTrigger(BaseTrigger):
     :param task_id: The task_id that contains the task you want to
         wait for. If ``None`` (default value) the sensor waits for the DAG
```

Comment:
The statement "If ``None`` the sensor waits for the DAG" is untrue, no such behavior is currently implemented in this Trigger